packages/dso-toolkit/src/styles/components/_navs.scss

```diff
@@ -20,7 +20,7 @@ $dso-navigation-selected-color: $cannon-pink;
       &,
       &:hover,
       &:focus,
-      &:visited {
+      &#{$not-dso-buttons}:visited {
```

Comment:
Ook hier: dit ziet er toch eigenlijk heel jammer uit...? Zouden we toch niet beter daar waar die paarse `:visited` nu blijkbaar meer gewicht krijgt dan vroeger, wat specifieker kunnen worden, in plaats van op alle anti-plekken weer meer gewicht in de schaal erbij te leggen?

Reply:
Ik denk niet dat dat eenvoudig gaat. Er is op heel hoog niveau besloten dat "a's, maar niet a's die een knop zijn" de scampi kleur krijgen, en elke ":not()' daarin geeft een dikke punt gewicht.

Deze constructie, hoewel het er raar uit ziet, geeft eenzelfde regel terug, maar met meer scoping op context. De andere mogelijkheid zou een `!important` zijn, maar we weten hoe we daar over denken.

Reply:
![image](https://user-images.githubusercontent.com/5871330/131986114-930f5d8b-c589-4168-bcdf-c75a015055f6.png)

Reply:
Yep I know. Maar ip voel me er superslecht bij. Voor dit geval zou ik bijna toch stellen: we halen die hele bak `:not()` eraf, en stylen juist expliciet de `:visited` kleur op de WEL #$dso-buttons situatie...

Reply:
In https://github.com/dso-toolkit/dso-toolkit/pull/1267 doen we een compleet andere approach